### DISCUSSION TITLE
Scrollbar in editor right-click menu

### DISCUSSION THREAD

**Discussion by nexensys** [ScratchAddons/ScratchAddons#2382] [Help]
**Describe the bug**

My script collapsing addon (which I added to my version of 14.0) now makes the right-click menu have a scrollbar.

**To Reproduce**

1. Download the addon's code [here](https://github.com/ErrorGamer2000/ScratchAddons/tree/Collapsible-Scripts/addons/script-collapse) and the l10n [here](https://github.com/ErrorGamer2000/ScratchAddons/blob/Collapsible-Scripts/addons-l10n/en/script-collapse.json).
2. Add files, reload extension and enable the addon.
3. Open the editor
4. Right click on a block that is not an operator, hat, or custom block.

**Expected behavior**

When the right-click menu pops up it should be the full height and have no scrollbar.

**Screenshots**

![image](https://user-images.githubusercontent.com/70162741/117154423-103ccc00-ad79-11eb-965f-5ffe7e84c148.png)

**Device**

- OS: Windows 10 OS Version 2009
- Browser: Chrome
- Version: 90.0.4430.93
- JavaScript: V8 9.0.257.23

Any ideas how to fix this?

**Comment by TheColaber** [comment]
Converted to a discussion because this is not scratch addons' fault. 
![image](https://user-images.githubusercontent.com/72760579/117156059-4edb8200-ad8b-11eb-83e2-a44323907e67.png)
Injecting items like this, which other addons also do is not the best way, which is why #1754 was made.

**Reply by nexensys** [reply]
Will you check over my revision? https://github.com/ScratchAddons/ScratchAddons/pull/2333/commits/ad237584922107555354f5760e9ef200d22ec337

**Reply by TheColaber** [reply]
LGTM... you can test it when my PR gets merged

**Reply by nexensys** [reply]
Okay, thanks.

**Reply by Endermaster3690** [reply]
> LGTM... you can test it when my PR gets merged

what does LGTM mean?

**Reply by TheColaber** [reply]
Ask google

**Reply by Endermaster3690** [reply]
> Ask google

aight thanks
